src/lib/Log.ts

```diff
@@ -0,0 +1,30 @@
+import crashlytics from '@react-native-firebase/crashlytics'
+import { logger, consoleTransport } from 'react-native-logs'
+
+const crashlyticsTransport = ({ msg }: { msg: string }) => crashlytics().log(msg)
+
+const config = {
+  transport: __DEV__ ? consoleTransport : crashlyticsTransport,
+  levels: {
+    trace: 0,
+    debug: 1,
+    info: 2,
+    warn: 3,
+    error: 4,
+  },
+  printLevel: false,
+  printDate: true,
+  severity: 'debug',
+}
+
+interface Logger {
```

Comment:
Voor het packagen moest er een interface worden aangemaakt voor het object wat geexporteerd ging worden.
Blijkbaar mag je geen private of protected variabelen exporteren en dat deden we:
```
 src/lib/Log.ts:28:7 - error TS4094: Property '_transportOptions' of exported class expression may not be private or protected.

28 const Log = logger.createLogger(config)
         ~~~
```

De variabele `_transportOptions` zit op het object dat wordt gecreeerd door `createLogger`.